demo/api/src/config/config.ts

```diff
@@ -17,6 +17,7 @@ export function createConfig(processEnv: NodeJS.ProcessEnv) {
         helmRelease: envVars.HELM_RELEASE,
         apiUrl: envVars.API_URL,
         apiPort: envVars.API_PORT,
+        adminUrl: envVars.ADMIN_URL,
```

Comment:
I wasn't sure if it's a good idea to add the ADMIN_URL to the api config. Otherwise we can just hardcode `http://localhost:*`